### PR TITLE
added nexus installation on ocp

### PR DESCRIPTION
**Pull request by xinredhat**

### CODE REVIEW

File: nexus/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default
```
Comment (prietyc123) [inline nexus/namespace.yaml:4]: I think its a typo 
```suggestion
  name: nexus
```

**Review by prietyc123 — CHANGES_REQUESTED** [review]

File: nexus/namespace.yaml (hunk 1 above)
Comment (xinredhat) [reply nexus/namespace.yaml:4]: no, it's not typo. when performing it , the namespace will be replaced by the namespace value defined in `kustomization.yaml`

Comment (prietyc123) [reply nexus/namespace.yaml:4]: Ahhh I didn't notice that. Thank you

**Pull request merged**